landing-page/landing-page.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title> thinzy </title>
+    <link rel="stylesheet" href="./landing-page.css">
+</head>
+
+<body>
+    <div class="wrapper-homepage">
+        <main>
+            <h1> thinzy </h1>
+            <h2> A <span class="homepage-span"> minimal </span> , open source, <span class="homepage-span"> ready-to-use </span> CSS component library </h2>
+            <a href="#0"> <button class="btn btn-primary btn-landing-page"> Get Started </button> </a>
```

Comment:
Why did you use 0 after # in href="#0"

Is there any specific reason?

Reply:
1. Hi Savita, use of "#"  will make the anchor tag blank. 
2. But sometimes we want to link some **section** of the same page with id **"learn"**. In that case we use id (as a selector). Link will become  "a href="#learn". 
3. Putting a '0' after "#" ensures that I have NOT forgotten the id there, but it is an empty link. This is important when other people are working with our code. 
4. It is an extra step to avoid confusion.

Reply:
OH, that is something new. Good. Thanks :-)